k8s/aat/money-claims/claim-store/claim-store.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: cmc-claim-store
+  namespace: cmc
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:latest-*
+spec:
+  releaseName: cmc-claim-store
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: cmc-claim-store
+    version: 0.1.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "4d7922fc-7d37-4278-b542-6ac8fe0408e2"
+      image: hmcts.azurecr.io/hmcts/cmc-claim-store:latest
+      ingressHost: cmc-claim-store.service.core-compute-aat.internal
+      keyVaults:
+        cmc:
+          resourceGroup: cmc
+          secrets:
+          - claim-store-db-password
```

Comment:
You will have to add other secrets as well in the list , citizen-oauth-client-secret
        - claim-store-s2s-secret
        - anonymous-caseworker-username
        - anonymous-caseworker-password
        - system-update-username
        - system-update-password
        - notify-api-key
As computed values will override the values in values.template.yaml instead of merging them

Reply:
i see what you mean - ill change